src/BloomExe/HtmlThumbNailer.cs

```diff
@@ -203,7 +203,7 @@ private Size SetWidthAndHeight(GeckoWebBrowser browser)
 				//Debug.Fail("Reproduction of BL-524: Crash making thumbnail?");
 				//otherwise, don't tell the user, just log and send exception if they're online
 				Logger.WriteEvent("***Error making thumbnail, possible bl-524 reproduction, swallowed. "+ e.Message);
-				Analytics.ReportException(e);
+				Analytics.ReportException(e, Program.GetProgramProperties());
```

Comment:
Seeing this change to calls to ReportException in half a dozen places makes me want a common method somewhere in Bloom for reporting exceptions to analytics. A static method on Program.cs might be appropriate, or wherever we initialize analytics.